From c34a5ac00c71da2f62455e43b0dde25530beaafd Mon Sep 17 00:00:00 2001
From: Michael Roth <mdroth@linux.vnet.ibm.com>
Date: Mon, 5 Mar 2012 17:44:07 -0500
Subject: [PATCH 40/98] guest agent: use QERR_UNSUPPORTED for disabled RPCs

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 9af99f1daf8e9bf0fecddcde35273383ec02cd45)
Signed-off-by: Jeff Cody <jcody@redhat.com>
Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qga/guest-agent-commands.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/qga/guest-agent-commands.c b/qga/guest-agent-commands.c
index e215bd3..624972e 100644
--- a/qga/guest-agent-commands.c
+++ b/qga/guest-agent-commands.c
@@ -521,7 +521,7 @@ static void guest_fsfreeze_cleanup(void)
  */
 GuestFsfreezeStatus qmp_guest_fsfreeze_status(Error **err)
 {
-    error_set(err, QERR_COMMAND_NOT_FOUND, "guest_fsfreeze_status");
+    error_set(err, QERR_UNSUPPORTED);
 
     return 0;
 }
@@ -532,7 +532,7 @@ GuestFsfreezeStatus qmp_guest_fsfreeze_status(Error **err)
  */
 int64_t qmp_guest_fsfreeze_freeze(Error **err)
 {
-    error_set(err, QERR_COMMAND_NOT_FOUND, "guest_fsfreeze_freeze");
+    error_set(err, QERR_UNSUPPORTED);
 
     return 0;
 }
@@ -542,7 +542,7 @@ int64_t qmp_guest_fsfreeze_freeze(Error **err)
  */
 int64_t qmp_guest_fsfreeze_thaw(Error **err)
 {
-    error_set(err, QERR_COMMAND_NOT_FOUND, "guest_fsfreeze_thaw");
+    error_set(err, QERR_UNSUPPORTED);
 
     return 0;
 }
-- 
1.7.7.6

